### ISSUE TITLE
Patch Error 

### ISSUE THREAD

**Issue by Giovannimbesi25** [yilozt/mutter-rounded#72]
I get this error while compiling the script ./package.sh

[run] patch -p1
patching file data/org.gnome.mutter.gschema.xml.in
patching file src/compositor/compositor-private.h
patching file src/compositor/compositor.c
Hunk #2 succeeded at 515 (offset 1 line).
Hunk #3 succeeded at 565 (offset 1 line).
Hunk #4 FAILED at 896.
Hunk #5 succeeded at 930 (offset 4 lines).
Hunk #6 succeeded at 1116 (offset 14 lines).
Hunk #7 succeeded at 1195 (offset 20 lines).
Hunk #8 succeeded at 1217 (offset 21 lines).
Hunk #9 succeeded at 1562 (offset 21 lines).
1 out of 9 hunks FAILED -- saving rejects to file src/compositor/compositor.c.rej
patching file src/compositor/meta-window-actor-private.h
patching file src/compositor/meta-window-actor-wayland.c
Hunk #2 succeeded at 135 with fuzz 1 (offset 5 lines).
Hunk #3 succeeded at 198 (offset 5 lines).
Hunk #4 succeeded at 509 (offset 5 lines).
Hunk #5 succeeded at 544 (offset 5 lines).
Hunk #6 succeeded at 564 with fuzz 1 (offset 5 lines).
Hunk #7 succeeded at 578 (offset 5 lines).
patching file src/compositor/meta-window-actor-x11.c
Hunk #2 succeeded at 459 (offset 20 lines).
Hunk #3 succeeded at 599 (offset 20 lines).
Hunk #4 succeeded at 758 (offset 20 lines).
Hunk #5 succeeded at 1102 (offset 20 lines).
Hunk #6 succeeded at 1114 (offset 20 lines).
Hunk #7 succeeded at 1222 (offset 20 lines).
Hunk #8 succeeded at 1246 (offset 20 lines).
Hunk #9 succeeded at 1397 (offset 20 lines).
patching file src/compositor/meta-window-actor.c
patching file src/core/prefs.c
patching file src/meson.build
Hunk #1 succeeded at 173 (offset -1 lines).
Hunk #2 succeeded at 474 (offset -1 lines).
Hunk #3 succeeded at 1030 (offset -1 lines).
patching file src/meta/prefs.h
patching file src/ui/frames.c
patching file src/ui/frames.h
patching file src/wayland/meta-window-wayland.c
[err: 1] patch -p1

**Comment by Enqueued** [comment]
Yea I'm getting the same error as well. 
It was working perfectly well for the past 2 days or so since I installed it then I got this error after an update. 

Running: 
Fedora 36  - Mutter version 42.2 - Gnome Shell version 42.2

**Comment by RoyalTwo** [comment]
Yeah, I think Mutter just updated to 42.2, and mutter-rounded is for 42.1

**Comment by Giovannimbesi25** [comment]
I don't know if it's related but, also Pop_os 22.04LTS, after the installation of mutter-rounded, notify me the for upgrades, which instal again mutter.

**Comment by ran-dall** [comment]
> Yeah, I think Mutter just updated to 42.2, and mutter-rounded is for 42.1

I can confirm that this is correct. There is a breaking commit in 42.2.

**Comment by yilozt** [comment]
It can be built in Fedora 36 for mutter 42.2 now. You can download rpm packages from [https://github.com/yilozt/mutter-rounded/actions/runs/2438198432](https://github.com/yilozt/mutter-rounded/actions/runs/2438198432)
